src/test/resources/hudson/plugins/active_directory/ActiveDirectorySecurityRealmTest/testFallBackUserDomain/config.xml

```diff
@@ -7,6 +7,8 @@
   <useSecurity>true</useSecurity>
   <authorizationStrategy class="hudson.security.AuthorizationStrategy$Unsecured"/>
   <securityRealm class="hudson.plugins.active_directory.ActiveDirectorySecurityRealm" plugin="active-directory@2.6">
+    <startTls>true</startTls>
+    <requireTLS>true</requireTLS>
```

Comment:
why is this configuration needed now?

Reply:
To make sure things are working fine with or without configs.

Reply:
Sorry but I don't understand this reasoning. This is actually even changing the initial configuration for the test. `null` will be as having `false`, while now it has `true`. If you had to add that because of your changes, please let us know the root cause, so we can understand the risks of having old configurations without those values

Reply:
I'll change it back to the initial configuration, this is not having any impact on the outcome of the test results.